includes/discovery/sensors/current/rfc1628.inc.php

```diff
@@ -34,6 +34,9 @@
     if (count($output_current) > 1) {
         $descr .= " Phase $index";
     }
+    if (is_array($data['upsOutputCurrent'])) {
```

Comment:
good, they messed up their rfc1628 implementation...

It would be better to check `isset($data['upsOutputCurrent'][0])` I think

Reply:
There were a few other instances of this check in rfc1628 code, I updated them too.